pkgs/development/libraries/jcal/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, which, libtool, getopt, readline}:
+
+stdenv.mkDerivation rec {
+  name = "jcal";
+  version = "v0.4.1";
+
+  src = fetchFromGitHub {
+    owner = "fzerorubigd";
+    repo = "jcal";
+    rev = "e4678bebb7efb7faa4435be3e87fc64da53e942d";
+    sha256 = "1b2nm499wcbpp2xim30b6sazkcg901cfr75qigclschj5rns6s9w";
+  };
+  doCheck = true;
+
+ # url = "https://github.com/fzerorubigd/jcal/";
```

Comment:
Why this comment block?

Reply:
Oh. i forgot to delete those lines of comments. So all commits can be removed.